diff --git a/src/schema.rs b/src/schema.rs
index a3404c7..9858b21 100644
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -67,12 +67,15 @@ table! {
         index_in_action_receipt -> Int4,
         action_kind -> Action_kind,
         args -> Jsonb,
         receipt_predecessor_account_id -> Text,
         receipt_receiver_account_id -> Text,
         receipt_included_in_block_timestamp -> Numeric,
+        is_delegate_action -> Bool,
+        delegate_parameters -> Nullable<Jsonb>,
+        delegate_parent_index_in_action_receipt -> Nullable<Int4>,
     }
 }
 
 table! {
     use diesel::sql_types::*;
 
@@ -121,13 +124,13 @@ table! {
 
 table! {
     use diesel::sql_types::*;
     use crate::models::enums::*;
 
     #[allow(non_snake_case)]
-    assets__fungible_token_events (emitted_for_receipt_id, emitted_at_block_timestamp, emitted_in_shard_id, emitted_index_of_event_entry_in_shard, emitted_by_contract_account_id, amount, event_kind, token_old_owner_account_id, token_new_owner_account_id, event_memo) {
+    assets__fungible_token_events (emitted_for_receipt_id, emitted_index_of_event_entry_in_shard) {
         emitted_for_receipt_id -> Text,
         emitted_at_block_timestamp -> Numeric,
         emitted_in_shard_id -> Numeric,
         emitted_index_of_event_entry_in_shard -> Int4,
         emitted_by_contract_account_id -> Text,
         amount -> Text,
@@ -140,13 +143,13 @@ table! {
 
 table! {
     use diesel::sql_types::*;
     use crate::models::enums::*;
 
     #[allow(non_snake_case)]
-    assets__non_fungible_token_events (emitted_for_receipt_id, emitted_at_block_timestamp, emitted_in_shard_id, emitted_index_of_event_entry_in_shard, emitted_by_contract_account_id, token_id, event_kind, token_old_owner_account_id, token_new_owner_account_id, token_authorized_account_id, event_memo) {
+    assets__non_fungible_token_events (emitted_for_receipt_id, emitted_index_of_event_entry_in_shard) {
         emitted_for_receipt_id -> Text,
         emitted_at_block_timestamp -> Numeric,
         emitted_in_shard_id -> Numeric,
         emitted_index_of_event_entry_in_shard -> Int4,
         emitted_by_contract_account_id -> Text,
         token_id -> Text,
@@ -156,13 +159,13 @@ table! {
         token_authorized_account_id -> Text,
         event_memo -> Text,
     }
 }
 
 table! {
-     use diesel::sql_types::*;
+    use diesel::sql_types::*;
     #[allow(non_snake_case)]
     aggregated__lockups(account_id) {
         account_id -> Text,
         creation_block_height -> Nullable<Numeric>,
         deletion_block_height -> Nullable<Numeric>,
     }
@@ -256,12 +259,15 @@ table! {
 
     transaction_actions (transaction_hash, index_in_transaction) {
         transaction_hash -> Text,
         index_in_transaction -> Int4,
         action_kind -> Action_kind,
         args -> Jsonb,
+        is_delegate_action -> Bool,
+        delegate_parameters -> Nullable<Jsonb>,
+        delegate_parent_index_in_transaction -> Nullable<Int4>,
     }
 }
 
 table! {
     use diesel::sql_types::*;
     use crate::models::enums::*;
